doc/classes/RichTextLabel.xml

```diff
@@ -341,14 +353,19 @@
 	</methods>
 	<members>
 		<member name="bbcode_enabled" type="bool" setter="set_use_bbcode" getter="is_using_bbcode">
+			Changes whether the label uses BBCode.
 		</member>
 		<member name="bbcode_text" type="String" setter="set_bbcode" getter="get_bbcode">
+			The BBCode text in the label.
 		</member>
 		<member name="override_selected_font_color" type="bool" setter="set_override_selected_font_color" getter="is_overriding_selected_font_color">
+			Use a custom select font color instead of the default color.
 		</member>
 		<member name="percent_visible" type="float" setter="set_percent_visible" getter="get_percent_visible">
+			Sets the text's visibility, and uses a floating point value between 0.0 and 1.0.
```

Comment:
We shouldn't use "Sets" on a member variable description, because it's used to both set and get the value.

"The text's visibility, as a [float] between 0.0 and 1.0."